Python/Distance_Calculator_App/README.md

```diff
@@ -0,0 +1,23 @@
+# Distance Calculating App
+Calculating distance between two locations is a basic requirement if you are working with raw location data. This app calculates distance between two geo-locations. You can get distance in kilometers between two locations just by entering names of the locations.
+
+## Quick Start:
+  
+- Change Directory
+
+``` cd .\Rotten-Scripts\Python\Distance_Calculator_App\ ```
+      
+- Install requirememnts
+
+`pip install -r requirements.txt`
+      
+- Run python file
+
+`python main.py --firstlocation <enter first location here> --secondlocation <enter second location here>`
+      
+## Screenshot
+
+<blockquote class="imgur-embed-pub" lang="en" data-id="a/fz13QZU" data-context="false" ><a href="//imgur.com/a/fz13QZU">Screenshot</a></blockquote>
```

Comment:
The image is not rendering in README, rather Screenshot is a hyperlink.

Kindly ensure that it enders on README